bin/messageformat.js

```diff
@@ -14,17 +14,14 @@ var fs = require('fs'),
     knownOpts = {
       help: Boolean,
       locale: [String, Array],
-      namespace: String
+      namespace: String,
+      'disable-plural-key-checks': Boolean
     },
     shortHands = {
       h: ['--help'],
       l: ['--locale'],
       n: ['--namespace']
```

Comment:
I considered a shorthand but since `-d` generally has to do with deletion, I couldn't think of a good one.